README.md

```diff
@@ -1,44 +1,117 @@
-## How to start
+## Instalation
```

Comment:
Please add instructions for add peerDependencies `styled-components`.
Because styled-components are required and this package is not automatically installed on this package.